src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/service/ReferralServiceTest.kt

```diff
@@ -99,10 +96,9 @@ class ReferralServiceTest @Autowired constructor(
   private val referralAccessChecker: ReferralAccessChecker = mock()
   private val userTypeChecker = UserTypeChecker()
   private val serviceProviderAccessScopeMapper: ServiceProviderAccessScopeMapper = mock()
-  private val referralAccessFilter: ReferralAccessFilter = mock()
+  private val referralAccessFilter = ReferralAccessFilter(serviceProviderAccessScopeMapper)
```

Comment:
the access filter is now completely responsible for filtering sent referrals for service providers - all the existing tests pass, as long as this is no longer mocked.